tests/integration/states/azurerm/containerregistry/test_registry.py

```diff
@@ -0,0 +1,75 @@
+import pytest
+
+
+@pytest.mark.second
+@pytest.mark.asyncio
+async def test_present(hub, ctx, resource_group, location, acr):
+    expected = {
+        "changes": {
+            "new": {
+                "admin_user_enabled": False,
+                "name": acr,
+                "resource_group": resource_group,
+                "sku": "Basic",
+                "replica_locations": [],
+            },
+            "old": {},
+        },
+        "comment": f"Container registry {acr} has been created.",
+        "name": acr,
+        "result": True,
+    }
+    ret = await hub.states.azurerm.containerregistry.registry.present(
+        ctx, acr, resource_group, sku="Basic", location=location,
+    )
+    assert ret == expected
+
+
+@pytest.mark.run(after="test_present", before="test_absent")
```

Comment:
I would add "order=2" or mark as second here as well mainly for uniformity (the incoming integration tests all do that as well) and so that the tests in this category will be grouped closer together. This concept applies to all of the other tests you wrote as well.

Reply:
I'm not sold that it actually matters. As long as "changes" occurs between the creation and destruction of the resource, do we really care exactly when it runs?

Reply:
It is one of those things that depends on the resource. Sometimes changes can end up impacting the results of other tests and I found that adding it in helped prevent that. That being said, it seems like these will be fine without it.